frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelStepTable.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { FunnelLayout } from 'lib/constants'
+import { funnelLogic } from 'scenes/funnels/funnelLogic'
+import Table, { ColumnsType } from 'antd/lib/table'
+import { PHCheckbox } from 'lib/components/PHCheckbox'
+import { SeriesToggleWrapper } from 'scenes/insights/InsightsTable/components/SeriesToggleWrapper'
+import { formatBreakdownLabel } from 'scenes/insights/InsightsTable/InsightsTableV2'
+import { cohortsModel } from '~/models/cohortsModel'
+import { InsightLabel } from 'lib/components/InsightLabel'
+import { SeriesGlyph } from 'lib/components/SeriesGlyph'
+import { getSeriesColor, humanizeOrder } from 'scenes/funnels/funnelUtils'
+import { ValueInspectorButton } from 'scenes/funnels/FunnelBarGraph'
+import { humanFriendlyDuration, humanizeNumber } from 'lib/utils'
+import { FlattenedFunnelStep } from '~/types'
+
+interface FunnelStepTableProps {
+    layout?: FunnelLayout // Not yet implemented
+}
+
+function getStepColor(step: FlattenedFunnelStep, isBreakdown?: boolean): string {
+    return getSeriesColor(isBreakdown ? step.breakdownIndex : step.order) || 'var(--primary)'
```

Comment:
I think we default to `text-default` when colors run out

Reply:
Splitting this into two cases; one with fallback to `text-default`, another with `primary` (for the checkbox). Feels like the checkbox should default to blue.